spring-cloud-sleuth-core/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -30,6 +30,12 @@
       "description": "Enable custom HystrixConcurrencyStrategy that wraps all Callable instances into their Sleuth representative - the TraceCallable.",
       "defaultValue": true
     },
+    {
```

Comment:
Since you're adding a `ConfigurationProperty` you don't need to provide it here. It's enough to pass it in the javadocs of the class.